addons/l10n_account_edi_ubl_cii_tests/tests/test_xml_cii_us.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+
+from odoo.addons.l10n_account_edi_ubl_cii_tests.tests.common import TestUBLCommon
+from odoo.tests import tagged
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestCIIUS(TestUBLCommon):
+
+    @classmethod
+    def setUpClass(cls,
+                   chart_template_ref=None,
+                   edi_format_ref="account_edi_ubl_cii.edi_facturx_1_0_05",
+                   ):
+        super().setUpClass(chart_template_ref=chart_template_ref, edi_format_ref=edi_format_ref)
+
+        cls.partner_1 = cls.env['res.partner'].create({
+            'name': "partner_1",
+            'country_id': cls.env.ref('base.us').id,
+        })
+
+        cls.partner_2 = cls.env['res.partner'].create({
+            'name': "partner_2",
+            'country_id': cls.env.ref('base.us').id,
+        })
+
+    @classmethod
+    def setup_company_data(cls, company_name, chart_template):
+        # OVERRIDE
+        res = super().setup_company_data(
+            company_name,
+            chart_template=chart_template,
+            country_id=cls.env.ref("base.us").id,
```

Comment:
I'm not sure you need this since the COA is already putting the country on the company

Reply:
I just checked and it doesn't (I'm using the generic COA here)